packages/app-project/src/screens/ProjectHomePage/components/ZooniverseTalk/components/RecentSubjects/components/MessageBox/MessageBox.js

```diff
@@ -0,0 +1,19 @@
+import { Box } from 'grommet'
+import React from 'react'
+
+function MessageBox (props) {
+  return (
+    <Box
+      align='center'
+      animation={['fadeIn', 'slideUp']}
+      background='light-1'
```

Comment:
Not sure if dark theme is in scope of the initial PR, but this should have a dark theme variant. Doesn't look great at the moment with dark theme: 

<img width="1402" alt="Screen Shot 2019-07-29 at 11 58 34 AM" src="https://user-images.githubusercontent.com/5016731/62067565-a71a4300-b1f9-11e9-9163-9b6a961e4480.png">

Reply:
<img width="1366" alt="Screenshot 2019-07-30 12 20 15" src="https://user-images.githubusercontent.com/2725841/62125212-6cc9a800-b2c4-11e9-9637-35f56c8e1267.png">